infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -0,0 +1,172 @@
+  APIM_MDM_VALUE: 'test-value' // different in staging and dev
+}
+
+// These values are taken from an export of Configuration on Dev
+// TODO:FN-419 Add to GitHub secrets?
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'  // different in staging and dev
+  MACHINEKEY_DecryptionKey: 'test-value' // different in staging and dev
+  MULESOFT_API_UKEF_MDM_EA_KEY: 'test-value'
+  MULESOFT_API_UKEF_MDM_EA_SECRET: 'test-value'
+  MULESOFT_API_UKEF_MDM_EA_URL: 'test-value' // different in staging and dev
+}
+
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/azure-function-acbs:${environment}'
+
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' existing = {
+  name: storageAccountName
+}
+
+var storageAccountKey = storageAccount.listKeys().keys[0].value
+
+// These values are hardcoded in the CLI scripts
+var settings = {
+  FUNCTIONS_WORKER_RUNTIME: 'node'
+  // TODO:FN-419 consider making this a parameter
+  WEBSITE_DNS_SERVER: '168.63.129.16'
+  WEBSITE_VNET_ROUTE_ALL: '1'
+}
+
+// These values are taken from an export of Configuration on Dev
+var additionalSettings = {
+  APPINSIGHTS_INSTRUMENTATIONKEY: applicationInsights.properties.InstrumentationKey
+  // TODO:FN-419 - Exported string had "EndpointSuffix=core.windows.net" in it. Check if needed
+  AzureWebJobsStorage: 'DefaultEndpointsProtocol=https;AccountName=${storageAccountName};AccountKey=${storageAccountKey}'
+  // TODO:FN-419 I think DOCKER_CUSTOM_IMAGE_NAME is overridden by linuxFxVersion
+  DOCKER_CUSTOM_IMAGE_NAME: dockerImageName
+  DOCKER_ENABLE_CI: 'true'
+  DOCKER_REGISTRY_SERVER_URL: '${containerRegistryName}.azurecr.io'
+  DOCKER_REGISTRY_SERVER_USERNAME: dockerRegistryServerUsername
+  FUNCTION_APP_EDIT_MODE: 'readOnly'
+  FUNCTIONS_EXTENSION_VERSION: '~3'
+  LOG4J_FORMAT_MSG_NO_LOOKUPS: 'true'
+  TZ: 'Europe/London'
+  WEBSITE_USE_PLACEHOLDER: '0'
+  WEBSITES_ENABLE_APP_SERVICE_STORAGE: 'false'
+}
+
+var nodeEnv = environment == 'dev' ? {NODE_ENV: 'development'} : {}
+
+var appSettings = union(settings, secureSettings, additionalSettings, additionalSecureSettings, nodeEnv)
+
+var functionAcbsName = 'tfs-${environment}-function-acbs'
+var privateEndpointName = 'tfs-${environment}-function-acbs'
+var applicationInsightsName = 'tfs-${environment}-function-acbs'
+
+
+// Minimal setup from MS example
+// See also https://learn.microsoft.com/en-my/azure/azure-functions/functions-infrastructure-as-code?tabs=bicep
+
+resource functionAcbs 'Microsoft.Web/sites@2022-09-01' = {
+  name: functionAcbsName
+  location: location
+  tags: {
+    Environment: 'Preproduction'
+  }
+  kind: 'functionapp,linux,container'
+  properties: {
+    httpsOnly: false
+    serverFarmId: appServicePlanId
+    siteConfig: {
+      // TODO:FN-419 These siteConfig values are included in the much more comprehensive config object, produced by the export.
```

Comment:
Note that when I re-export this, I get the duplication, so I think all is well and I put the settings here, and just use the `functionAcbsAppSettings` resource below for environment variables.